docker/ds/scripts/backup.sh

```diff
@@ -8,18 +8,35 @@ source /opt/opendj/env.sh
 
 #DATESTAMP=`date "+%Y/%m/%d"`
 
-mkdir -p "$BACKUP_DIRECTORY"
-chmod 775 "$BACKUP_DIRECTORY"
+mkdir -p "${BACKUP_DIRECTORY}"
```

Comment:
this image will soon be deprecated and moved to the ds repo (owned by ds team). So heads up